vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -44,29 +45,34 @@ def _get_id(site: dict) -> str:
     return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
 
 
+def _cleanup_url(url):
+    if not url:
+        return None
+    # if "@" in url:
+    #     # Some of these are email addresses.
+    #     # Skipping those for now.
+    #     return None
+    if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
+        return None
+    print(url)
+
+    url = re.sub(r"^https/:", "https://", url)  # fix typos
+    url = re.sub(r"^https:/t", "https://t", url)  # fix typos
+
+    # if not url.startswith("http"):
```

Comment:
This was probably fixing urls like` www.vaccine.org` so we should keep that?

Reply:
i suspect that the common utility normalizer handles these cases already

Reply:
Then just remove this commented out code